src/main/java/module/lineup/assistant/LineupAssistant.java

```diff
@@ -53,159 +53,250 @@ public final boolean isPlayerInStartingEleven(int spielerId, Vector<MatchLineupP
-			doPlayerLineupIdealPosition(IMatchRoleID.FORWARD_TOWING, bForm, bInjured, bSuspended, lPlayers, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.KEEPER, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.CENTRAL_DEFENDER, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.CENTRAL_DEFENDER_TOWING, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.CENTRAL_DEFENDER_OFF, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.BACK, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.BACK_TOMID, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.BACK_OFF, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.BACK_DEF, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.MIDFIELDER, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.MIDFIELDER_OFF, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.MIDFIELDER_DEF, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.MIDFIELDER_TOWING, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.WINGER, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.WINGER_OFF, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.WINGER_DEF, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.WINGER_TOMID, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.FORWARD, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.FORWARD_DEF, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
+			doPlayerLineupIdealPosition(IMatchRoleID.FORWARD_TOWING, bForm, ignoreInjured, ignoreSuspended, players, lPositions);
 		}
 
-		var fieldPlayerPositionOrder = new ArrayList<Byte>();
-		var reservePositionOrder = new ArrayList<Byte>();
-		fieldPlayerPositionOrder.add(IMatchRoleID.KEEPER);
-		reservePositionOrder.add(IMatchRoleID.KEEPER);
-		// nun reihenfolge beachten und unbesetzte füllen
+		var fieldPlayerPositionOrder = new ArrayList<List<Byte>>();
+		var reservePositionOrder = new ArrayList<List<Byte>>();
+		fieldPlayerPositionOrder.add(List.of(IMatchRoleID.KEEPER));
+		reservePositionOrder.add(List.of(IMatchRoleID.KEEPER));
 		switch (sectorsStrengthPriority) {
```

Comment:
Do you think it'd be possible some day to compute all 6 combinations in parallel, and identify the best approach for all 3 sectors?  Or better even let the user select _a posteriori_ what combination to pick for whatever sector they want maximised.

That would have a significant impact on computation, but also on UI, I think.

Reply:
Add "Auto" Entry to the selection list  - good idea